src/xcode/ENA/ENA/Source/View Helpers/__tests__/CountdownTimerTests.swift

```diff
@@ -11,6 +11,7 @@ class CountdownTimerTests: XCTestCase {
 	private var countdownTimerTarget: CountdownTimerTarget!
 
 	override func setUp() {
+		super.setUp()
```

Comment:
I was wondering if this is required. It is :)
https://developer.apple.com/forums/thread/113314?answerId=348312022#348312022